native_client/java/AndroidTest/deepspeechandroid/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+        minSdkVersion 15
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        ndk {
+            abiFilters("arm64-v8a", "armeabi-v7a", "x86_64") // Supported ABIs
```

Comment:
This is going to break our support handling different ABIs

Reply:
Aren't those the only ABIs that DeepSpeech supports?

Reply:
For now. And we avoid to hard-code the list directly in the `build.gradle` file to make it easier to integrate with CI tooling.